client_src/js/printerManagementRunner.js

```diff
@@ -3009,7 +3014,12 @@ class dashUpdate {
           }
         }
       } catch (e) {
-        console.log(e);
+        //TODO: Rework this file, this is dangerous and stops any error messages been able to be read without crashing the console...
+        // patched for now so only triggeres once encase we do encounter errors...
+        if (!sseErrorMessageTriggered) {
```

Comment:
Nice solution, could you create an issue for cleaning this up? Seems important to be fixed

Reply:
Yeah, the file needs a similar clean up as settings got tbh, will do